pkg/k8sapi/k8sutils.go

```diff
@@ -5,17 +5,39 @@ import (
 
 	eniconfigscheme "github.com/aws/amazon-vpc-cni-k8s/pkg/apis/crd/v1alpha1"
 	"github.com/aws/amazon-vpc-cni-k8s/pkg/utils/logger"
+	v1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/client-go/kubernetes"
 	clientgoscheme "k8s.io/client-go/kubernetes/scheme"
+	corev1 "k8s.io/client-go/kubernetes/typed/core/v1"
 	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/record"
 	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/cache"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
 var log = logger.Get()
 
+var eventRecorder record.EventRecorder
+
+func InitRecorder() {
+	restCfg, err := ctrl.GetConfig()
```

Comment:
Can we reuse GetKubeClientSet and remove these 8 lines?

Reply:
GetKubeClient was using "InClusterConfig" that was causing an issue earlier; but moving the recorder to ipamd seems to work. 
Changed it, thanks!